kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -576,9 +576,16 @@ define([
         }
 
         getJobInitCode() {
+            const currentCells = Jupyter.notebook.get_cells();
             return [
                 'from biokbase.narrative.jobs.jobcomm import JobComm',
-                'JobComm().start_job_status_loop(init_jobs=True)',
+                'cell_list = ' +
+                    JSON.stringify(
+                        currentCells.map((cell) => {
+                            return cell.cell_id;
+                        })
+                    ),
+                'JobComm().start_job_status_loop(cell_list=cell_list, init_jobs=True)',
```

Comment:
start the job status loop with a list of cell IDs in the current narrative

Reply:
Those are the Jupyter-made cell ids, not the ones that live in KBase metadata. Those would be under `cell.metadata.kbase.attributes.id`. Jupyter cell ids aren't always the same on restart. Unless that's what you mean?

Reply:
The KBase cell ids are what are sent along with job metadata.

Reply:
Doh... why have one unique cell ID when you could have several?! OK, will fix.